cmd/aws-cloud-controller-manager/main.go

```diff
@@ -107,28 +109,74 @@ func main() {
 				}
 			}
 
-			printFlags(cmd.Flags())
+			cliflag.PrintFlags(cmd.Flags())
 
-			c, err := s.Config(KnownControllers(), ControllersDisabledByDefault.List())
+			c, err := s.Config(KnownControllers(), app.ControllersDisabledByDefault.List())
```

Comment:
@cici37 most of the change here is referring to [sample main](https://github.com/kubernetes/cloud-provider/blob/master/sample/basic_main.go), can you help to review it and see if it is expected? Thanks!

Reply:
Looks good to me :)